frontend/src/app/core/apiv3/endpoints/work_packages/api-v3-work-packages-paths.ts

```diff
@@ -126,7 +126,7 @@ export class ApiV3WorkPackagesPaths extends ApiV3Collection<WorkPackageResource,
     const params = {
       sortBy: '[["updatedAt","desc"]]',
       offset: '1',
-      pageSize: '10',
+      pageSize: '20',
```

Comment:
This method is used in other places as well where we might not want 20 results. And nobody will know, that the 20 is specific to the search result. Instead of chaning it here, just pass `{ pageSize: "20" }` as paramter when calling the method.